blockchain/genesis/genesis.go

```diff
@@ -113,15 +113,21 @@ func defaultConfig() Genesis {
 	}
 }
 
-func initTestDefaultConfig() {
-	Default = defaultConfig()
-	Default.PacificBlockHeight = 0
+// TestDefault is the default genesis config for testing
+func TestDefault() Genesis {
+	ge := defaultConfig()
+	initTestDefaultConfig(&ge)
+	return ge
```

Comment:
`make test`, and use `TestDefault()` wherever the test fails

Reply:
I think it can be changed in another pr. This pr is to fix receipt issue.